### PR TITLE
Carousel implementation-after demo

### PR DESCRIPTION
**Pull request by DarinaBlazevica**
Carousel implementation after Demo
-CSS Transition and Transition group removed
-Animation bug removed

### CODE REVIEW

File: src/components/organisms/Carousel/Carousel.scss [hunk 1]
```diff
@@ -1,148 +1,118 @@
...
-
-.carousel__slider__text-slider .subheading--oranges {
-  top: 85px;
-  padding-right: 100px;
-  left: 18px;
-  position: absolute;
+.carousel-text-container__text-slider .subheading--oranges {
+  top:35px;
   text-transform: lowercase;
   font-weight: 550;
```
Comment (VSlobolinskyi) [inline src/components/organisms/Carousel/Carousel.scss]: This won't work. Font-weight for non-variable fonts has to be defined with step of a 100 units

File: src/components/organisms/Carousel/Carousel.scss [hunk 2]
```diff
@@ -1,148 +1,118 @@
...
-.slide-left-exit-active {
-  transform: translateX(100%);
-  transition: 1000ms ease-in-out;
-}
-
-.slide_text-right-enter {
-  transform: translateX(100%);
+.carousel-text-container__text-slider section {
+  width: calc(100% / 3);
+  flex-shrink: 0;
```
Comment (VSlobolinskyi) [inline src/components/organisms/Carousel/Carousel.scss]: Why do you need flex-shrink here?

**Review by ReinisZogots — APPROVED** [review]

File: src/components/organisms/Carousel/Carousel.scss (hunk 2 above)
Comment (DarinaBlazevica) [reply src/components/organisms/Carousel/Carousel.scss]: Mistake, changed

File: src/components/organisms/Carousel/Carousel.scss (hunk 1 above)
Comment (DarinaBlazevica) [reply src/components/organisms/Carousel/Carousel.scss]: Changed

File: src/App.scss [hunk 3]
```diff
@@ -14,7 +14,7 @@
 $axis: "Axis";
 $subheading-color: #985e5b;
 $font-size-subheading: 12px;
-$font-weight: 700;
+$font-weight: 100;
```
Comment (DarinaBlazevica) [inline src/App.scss:17]: Deleted this values(font-weight) in next commit, it's unnecessary at this place

**Review by VSlobolinskyi — APPROVED** [review]

**Pull request merged**